akka-docs/src/main/paradox/typed/persistence.md

```diff
@@ -33,7 +33,7 @@ This module is currently marked as @ref:[may change](../common/may-change.md) in
 
 ## Example
 
-Let's start with a simple example. The minimum required for a `PersistentBehavior` is:
+Let's start with a simple example. The minimum required for a `EventSourcedBehavior` is:
```

Comment:
I think we should re-title this to "Event Sourcing", WDYT? 

We can later add a persistence docs section which links to this and how to do async / blocking DB access from an actor

Or just in the intro saying that there is special support for event sourcing and regular database access just has to be careful about blocking.

Reply:
Re-reading maybe we shouldn't rename, we co call out it is for event sourcing already.